tests/integration/testlib/assertlib.py

```diff
@@ -53,3 +54,17 @@ def assert_absent(*ifnames):
 
     current_state = statelib.show_only(ifnames)
     assert not current_state[INTERFACES]
+
+
+def iface_macs(state):
+    for ifstate in state[Interface.KEY]:
+        yield ifstate[Interface.MAC]
+
+
+def assertMacAddress(state, expected_mac=None):
```

Comment:
I don't like `CamelCaseFunctionName` especial all other function are named as `fixing_something`.

Reply:
Neither do I. Named it that since Edy specifically had told me to do so

Reply:
@EdDev Are you suggesting we should use CamelCase as function name?

Reply:
It’s an old habit from the ‘unittest’ asserts. Use whatever you like.